lib/ansible/modules/storage/netapp/na_elementsw_drive.py

```diff
@@ -0,0 +1,321 @@
+        choices: ['present', 'absent', 'clean']
+        default: 'present'
+
+    node_id:
+        description:
+        - ID or Name of cluster node.
+        required: true
+
+    force_during_upgrade:
+        description:
+        - Flag to force drive operation during upgrade.
+        type: 'bool'
+
+    force_during_bin_sync:
+        description:
+        - Flag to force during a bin sync operation.
+        type: 'bool'
+'''
+
+EXAMPLES = """
+   - name: Add drive with status available to cluster
+     tags:
+     - elementsw_add_drive
+     na_element_drive:
+       hostname: "{{ elementsw_hostname }}"
+       username: "{{ elementsw_username }}"
+       password: "{{ elementsw_password }}"
+       state: present
+       drive_id: scsi-SATA_SAMSUNG_MZ7LM48S2UJNX0J3221807
+       force_during_upgrade: false
+       force_during_bin_sync: false
+       node_id: sf4805-meg-03
+
+   - name: Remove active drive from cluster
+     tags:
+     - elementsw_remove_drive
+     na_element_drive:
+       hostname: "{{ elementsw_hostname }}"
+       username: "{{ elementsw_username }}"
+       password: "{{ elementsw_password }}"
+       state: absent
+       force_during_upgrade: false
+       node_id: sf4805-meg-03
+       drive_id: scsi-SATA_SAMSUNG_MZ7LM48S2UJNX0J321208
+
+   - name: Secure Erase drive
+     tags:
+     - elemensw_clean_drive
+     na_elementsw_drive:
+       hostname: "{{ elementsw_hostname }}"
+       username: "{{ elementsw_username }}"
+       password: "{{ elementsw_password }}"
+       state: clean
+       drive_id: scsi-SATA_SAMSUNG_MZ7LM48S2UJNX0J432109
+       node_id: sf4805-meg-03
+
+   - name: Add all the drives of a node to cluster
+     tags:
+     - elementsw_add_node
+     na_elementsw_drive:
+       hostname: "{{ elementsw_hostname }}"
+       username: "{{ elementsw_username }}"
+       password: "{{ elementsw_password }}"
+       state: present
+       force_during_upgrade: false
+       force_during_bin_sync: false
+       node_id: sf4805-meg-03
+
+"""
+
+
+RETURN = """
+
+msg:
+    description: Success message
+    returned: success
+    type: string
+
+"""
+import traceback
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_native
+import ansible.module_utils.netapp as netapp_utils
+
+HAS_SF_SDK = netapp_utils.has_sf_sdk()
+
+
+class ElementSWDrive(object):
+    """
+    Element Software Storage Drive operations
+    """
+
+    def __init__(self):
+        self.argument_spec = netapp_utils.ontap_sf_host_argument_spec()
+        self.argument_spec.update(dict(
+            state=dict(required=False, choices=['present', 'absent', 'clean'], default='present'),
+            drive_id=dict(required=False, type='str'),
+            node_id=dict(required=True, type='str'),
+            force_during_upgrade=dict(required=False, type='bool'),
+            force_during_bin_sync=dict(required=False, type='bool')
```

Comment:
No, the default is None.

If the value is set to False or True, this may trigger a modify if the current value is different.
If the value is None, no action to taken for this parameter.